{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/settings/templatetags/settings.py

```diff
@@ -1,14 +1,18 @@
 from django_jinja import library
 
 from ..utils import get_setting as get_setting_util
+from ..utils import get_setting_value
 
 
 @library.global_function
-def setting(key):
-    obj = get_setting_util(key=key, default=None)
-    if not obj:
+def setting(key, default=None):
+    value = get_setting_value(key=key, default=None)
+    if value is None:
+        # rather than 'return default or key' - means coverage isn't fooled?
+        if default is not None:
+            return default
         return key
```

Comment:
A minor point, but personally I've never seen the value in returning the settings key to the front-end. I know it can be useful to see what you need to add, but I feel like we should be ensuring they're entered. I don't feel like the risk of having "facebook-url" output on the frontend is worth it.

Reply:
I can't muster an opinion either way - I would not mind removing this at all if you think that's the right thing.

Reply:
Come to think of it: Given that we have a `default` parameter in case we really want to see SOMETHING on the front end - yeah, let's go for it.